Toolset/palettes/revdatagridlibrary/behaviorsdatagridbuttonbehavior.livecodescript

```diff
@@ -210,12 +195,34 @@ private command _Initialize
       unlock messages
       
       put true into sInit
+   else
```

Comment:
@trevordevore thanks for this. It looks good the only issue is I think this needs to be guarded against execution on non-mobile platforms. So `else if the environment is "mobile" then`

Reply:
Fixed.